### PR TITLE
fix deb deps

### PR DESCRIPTION
**Pull request by alecGraves**

### CODE REVIEW

**Comment by alecGraves** [comment]
Only those with write access to this repository can merge pull requests.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,21 @@
+ros-melodic-realsense2-camera (92.2.13-0ubuntu) bionic; urgency=medium
```
Comment (seebq) [inline debian/changelog:1]: I've never seen this before, is this a good best practice?

File: debian/control [hunk 2]
```diff
@@ -15,7 +16,9 @@ Depends: ${shlibs:Depends}, ${misc:Depends},
  ros-melodic-cv-bridge, ros-melodic-nav-msgs, ros-melodic-nodelet,
  ros-melodic-roscpp, ros-melodic-genmsg, ros-melodic-sensor-msgs,
  ros-melodic-message-runtime, ros-melodic-tf, ros-melodic-std-msgs,
- ros-melodic-diagnostic-updater, libimage-transport-dev
+ ros-melodic-diagnostic-updater, libimage-transport-dev,
+ librealsense2-dkms (= 1.3.13-0ubuntu1),
```
Comment (seebq) [inline debian/control:20]: Thank you.

**Review by seebq — APPROVED** [review]
Cool, so we maintain this library ourselves because they don't debianize and publish yes?

Crazy question: could we do this for them and offer to debianize and host so that others might benefit from our debianization and package builds?

File: debian/changelog (hunk 1 above)
Comment (alecGraves) [reply debian/changelog:1]: idk, it is probably better to rename the package to something like greenzie-ros-realsense2-camera when building the deb. 

But this likely works and is 'idiot proof' as opposed to me renaming the package in all the right spots in our code and this code :stuck_out_tongue_closed_eyes:

File: debian/control (hunk 2 above)
Comment (alecGraves) [reply debian/control:20]: :+1:

**Comment by alecGraves** [comment]
https://github.com/Greenzie/robotic-lawnmower/pull/1685 goes into more detail.

@seebq we are doing this so we can use the kernel modules (librealsense2-dkms), which greatly reduces cpu load for data transfer from our 3 camera setup.

The community-maintained one uses a more portable librealsense2 package which does not use kernel modules. 

Our reasons for this are pretty much just performance, and I am not sure exactly who is maintaining the community ros package. We could offer to maintain and introduce our package as an alternative to the ros community, but giving the people a choice between two packages could just introduce user complexity which is unnecessary in most situations.

And also I intend to not update this package unless it fixes a bug or we want new features upstream. So I do not know if it is worth sharing.

**Comment by alecGraves** [comment]
And this really does not introduce anything that the community does not already have. If they add intel's librealsense2-dkms package and build this from source, they accomplished what is in this repo (just an extra manual step).

It might save people some time, but is largely only required due to our setup.

**Comment by seebq** [comment]
Got it. Great explanation, thank you!

File: realsense2_camera/CMakeLists.txt [hunk 3]
```diff
@@ -34,12 +34,17 @@ if(SET_USER_BREAK_AT_STARTUP)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBPDEBUG")
 endif()
 
-find_package(realsense2 2.29.0)
+find_package(realsense2 2.32.1)
```
Comment (seebq) [inline realsense2_camera/CMakeLists.txt:37]: I don't understand why this is here, is this a typo?  Shouldn't this be 2.33.1 ?

**Comment by seebq** [comment]
Closing in favor of #2 which is the same thing, but goes back to same versioning as upstream, which we now control.

**Pull request closed without merging**